sourcegraph/cody
# Cody Web: Remove implicit context in cody web

Closes https://linear.app/sourcegraph/issue/SRCH-820/make-cody-web-context-explicit

Prior to this PR, Cody Web used the initial context repository as a default source of context files for prompting. This raised some concerns about implicit context fetching, so this PR remotes this feature, and now chat relies only on user-provided file/repo/urls mentions in the message textbox. 

## Test plan
- Run Cody Web demo (`cd web` and `pnpm dev`)
- Check that when you don't mention any files repositories or URLs you don't have any context in chat UI as you submit the message 
- Check that mention context still works (try to mention some files or repos and check that context is pulled properly)



## Changelog
